_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+
+1. Is there a path to go by train?
+2. How far can you go in one day?
+3. Which stops are interesting?
+
+### Tickets or Interrail
+
+Usually people buy tickets from A to B, sometimes by locking in on a single connection.
+However, a travel of this size is different.
+
+When traveling huge distances over many countries sometimes delays happen or a train might break down.
+With a regular ticket you have to find staff and organize a reroute on the spot.
+
+This is where Interrail saves the day.
+With the Interrail global pass you buy travel days over a period of time.
+So you can simply catch the next train and continue your journey.
+
+Eg. buy 7 travel days within one months.
+So you have the flexibility to ride any train outside of your home country on 7 days.
+In addition, the ticket is also valid for travel in your country of residence during one outbound journey and one inbound journey that occur during travel days at any point in your trip.
+
+Most high-speed and night trains require a [reservation at an additional cost](https://www.interrail.eu/en/plan-your-trip/reservations).
+
+For most European citizens [the Interrail pass (currently 10% off)](https://www.interrail.eu/en/autumn-sale/global-pass) is probably the most flexible and affordable ticket.
+For [Non-EWR citizens](https://www.interrail.eu/en/help/ordering-pass/do-i-need-interrail-or-eurail-pass) there is [Eurail](https://www.eurail.com/en/eurail-passes/global-pass).
+
+### Tools
+
+Inside Spain, Portugal, France:
+* [trenes](https://train.trenes.com/) <- Don't forget to uncheck the "booking.com" button
+
+For northern Europa:
+* Austria, Germany, Czech Republic, Slovakia, Hungary, Italy, Switzerland, Serbia -> [ÖBB](https://www.oebb.at/)
+* Sweden, Denmark, Netherlands, Germany, Switzerland -> [Deutsche Bahn](https://bahn.de)
+* Switzerland, France, Spain -> [Schweizerische Bundesbahnen](https://sbb.ch)
+
+## The travel to Barcelona
+
+Now let us look at some examples:
+
+### Within Spain
+
+From Madrid to Barcelona there are two kinds of trains traveling.
+
+* A cheaper train that takes 9.5 hours.
+* With the high speed you can reach Barcelona in 2.5 hours.
+
+* From Sevilla Santa Justa you travel directly to Barcelona within 5.5 hours.
+
+### From France or Portugal
+
+From most bigger cities of France you can reach Barcelona within 7 to 9 hours.
+
+* From Paris, there is a direct TGV connection 3 times a day.
+* From Rennes, take the TGV via Nimes.
+* From Bordeaux, with InterCity and TGV via Narbonne.
+
+* From Lisboa Estacion Oriente there is a train to Madrid Chamartin. Since it is not a high speed train it takes 11 hours and you will probably need to continue on the second day.
+
+Note that there is a [cheaper Interrail country pass](https://www.interrail.eu/en/interrail-passes/one-country-pass/spain-passes) and you probably only need 3 days within 1 month.
+
+
+### From the North
+
+These events are timed so that people traveling towards Barcelona can stop by.
```

Comment:
It would make sense to mention that this is your route and your taking over planning this.

Reply:
Also state that this trip is intended to start Wednesday morning and you arrive in Barcelona by Thursday night.